.github/workflows/check-publish.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         with:
           target: wasm32-unknown-unknown
           components: rust-src
-          toolchain: nightly-2024-01-01
+          toolchain: nightly-2024-07-25
```

Comment:
update from `nightly-2024-01-01` since cargo stylus cannot compile on the previous version. Temporarily